app-header/app-header.html

```diff
@@ -475,24 +532,32 @@
        */
       _transformHeader: function(y) {
         this.translate3d(0, (-y) + 'px', 0);
+
         if (this._primaryEl && this.condenses && y >= this._primaryElTop) {
+
           this.translate3d(0, (Math.min(y, this._deltaHeight) - this._primaryElTop) + 'px', 0,
               this._primaryEl);
+
         }
       },
 
       _resizeHandler: function() {
         this.resetLayout();
-      }
+      },
 
       /**
-       * Fires when the `app-header` is transformed. That is, when scroll effects are taking place.
-       * This event doesn't bubble.
-       * * `event.detail.top` The top position relative to the viewport.
-       * * `event.detail.progress` The progress value.
+       * Returns an object containing the progress value of the scroll effects
+       * and the top position of the header.
        *
-       * @event app-header-transform
+       * @method getScrollState
+       * @return {Object}
        */
+      getScrollState: function() {
```

Comment:
yes, we are removing the event `app-header-transform`. Therefore, enabling this method would allow to lazily read the scroll state and get by without that event.